TextileIPFSNativeModule.js

```diff
@@ -103,5 +108,10 @@ export default {
   getAccessToken: async function (): string {
     const result = await TextileIPFS.getAccessToken()
     return result
-  }
+  },
+
+  eventEmitter: Platform.select({
+    ios: new NativeEventEmitter(Events),
+    android: DeviceEventEmitter,
+  })
```

Comment:
ios and android do this differently, this is taken from the article linked to in PR description